README.md

```diff
@@ -207,6 +207,14 @@ AutoImport({
     }
   ],
 
+  // Generate corresponding .eslintrc-auto-import.json file.
+  // eslint globals Docs - https://eslint.org/docs/user-guide/configuring/language-options#specifying-globals
+  eslintrc: {
+    enabled: false, // Default enabled when `eslint` is installed locally.
```

Comment:
> Default enabled when `eslint` is installed locally.

I am not very sure about this, as not all eslint users need this rule (e.g. I use TypeScript, which already has its own check for undeclared vars, so I disabled the eslint rule). I think it's better to have it default to `false`.

Reply:
👌